skbio/draw/tests/test_distributions.py

```diff
@@ -12,7 +12,10 @@
 
 import numpy as np
 import numpy.testing as npt
-import matplotlib.pyplot as plt
+try:
```

Comment:
I added a comment for future reference! Not sure if hiding it behind some sort of context manager could make it simpler or more maintainable.

OTOH, this made me think that this approach of trying to import optional dependencies on top, and passing in the except clause looks much neater and easier to maintain than repeating imports in every single method/function that needs them... Thoughts?